lib/manageiq/appliance_console/message_configuration_client.rb

```diff
@@ -75,6 +82,7 @@ def fetch_truststore_from_server
 
       def deactivate
         remove_installed_files
+        configure_messaging_type("miq_queue") # Settings.prototype.messaging_type = 'miq_queue'
```

Comment:
Should we change the messaging type first, restart the server then remove the installed files?

Reply:
Good point.
How about this:
```
stop_evmserverd # New method
remove_installed_files
configuremessaging_type("miq_queue")
restart evmserverd
```

Reply:
Addressed in commit https://github.com/ManageIQ/manageiq-appliance_console/pull/137/commits/42884bb4877b494dfde0e2a06e3f25dc139a17d2